mycore-restapi/src/main/java/org/mycore/restapi/MCRSessionFilter.java

```diff
@@ -314,12 +315,15 @@ public boolean isUserInRole(String role) {
 
         @Override
         public String getUserAttribute(String attribute) {
-            if (MCRUserInformation.ATT_REAL_NAME.equals(attribute)) {
-                return jwt.getClaim("name").asString();
-            }
-            if (MCRUserInformation.ATT_EMAIL.equals(attribute)) {
-                return jwt.getClaim("email").asString();
+            if (!jwt.getClaim(attribute).isNull()) {
+                return jwt.getClaim(attribute).asString();
```

Comment:
Not all attribute names are valid claim names. We should not mix this and be more careful: https://auth0.com/docs/tokens/json-web-tokens/json-web-token-claims

Reply:
Oh, right. I will additionally extend the claim names with a prefix.